docs/api.md

```diff
@@ -391,6 +391,30 @@ function sync() {
 
 Here we're solving encountered conflicts by picking all remote versions. After conflicts being properly addressed, we're syncing the collection again, until no conflicts occur.
 
+## The case of a new/flushed server
+
+In case a pristine new server is used, or one that has just been [flushed](http://kinto.readthedocs.org/en/latest/configuration/settings.html?highlight=flush#activating-the-flush-endpoint), [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync) will most probably reject with a *«Server has been flushed»* error. That means the remote server doesn't hold any data, while your local database is marked as synchronized and probably contains some data.
+
+So instead of wiping your local database to reflect this new remote state, you get a proper error :) In this situation, you'll probably want to republish your local database to the server; this is very easy to achieve by calling [`#resetSyncStatus()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-resetSyncStatus), then [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync) again:
+
+```js
+articles.sync()
+  .catch(err => {
+    if (err.message.contains("flushed")) {
+      return articles.resetSyncStatus()
+        .then(_ => articles.sync());
+    } else {
+      throw err;
+    }
+  })
+  .then(result => {
+    console.log(result);
+  })
+  .catch(error => {
+    console.error(error);
+  });
```

Comment:
nit: in other parts of the docs, `console.error.bind(console)` is used  (I personnaly prefer this form, but would recommand consistency).